hbase-http/src/main/java/org/apache/hadoop/hbase/http/HttpServer.java

```diff
@@ -115,6 +115,11 @@ public class HttpServer implements FilterContainer {
   // limitation otherwise the UTs will fail
   private static final int DEFAULT_MAX_HEADER_SIZE = Character.MAX_VALUE - 1;
 
+  //Add configuration for jetty idle timeout
+  private static final String HTTP_JETTY_IDLE_TIMEOUT = "hbase.master.ui.connection.idleTimeout";
```

Comment:
nit: this property actually affects all processes that run an http server, not only the master UI. So we should name it accordingly.